broker/src/crypto.rs

```diff
@@ -285,6 +285,20 @@ impl GetCerts for GetCertsFromPki {
             Ok(update) => update
         }
     }
+
+    async fn get_crl(&self) -> Result<Option<X509Crl>, SamplyBeamError> {
+        debug!("Getting crl");
+        let resp = self.resilient_vault_request(
+            &Method::GET,
+            &format!("{}/crl", self.pki_realm),
```

Comment:
The newer version of the api call is to fetch the issuer-specific CRL, cf. https://developer.hashicorp.com/vault/api-docs/secret/pki#read-issuer-crl
For now (and because we only have one issuer) this is ok